loop-client/README.md

```diff
@@ -0,0 +1,21 @@
+# Tests for Loop-Client
+
+##General Configuration
+
+It's highly recommended to use [Virtualenv](https://virtualenv.pypa.io/en/latest/)
+in order to have an isolated environment.
+
+From /loop-client/
+
+1. virtualenv .
```

Comment:
use: virtualenv venv
we're trying to standardize on that throughout our repo to prevent polluting the project env